test/src/test/java/org/apache/accumulo/test/util/CertUtilsTest.java

```diff
@@ -70,16 +72,18 @@ public void createSelfSigned() throws Exception {
   @Test
   public void createPublicSelfSigned() throws Exception {
     CertUtils certUtils = getUtils();
-    File tempSubDir = tempDir.toPath().resolve(testName()).toFile();
-    assertTrue(tempSubDir.isDirectory() || tempSubDir.mkdir());
-    File rootKeyStoreFile = tempSubDir.toPath().resolve("root.jks").toFile();
-    certUtils.createSelfSignedCert(rootKeyStoreFile, "test", PASSWORD);
-    File publicKeyStoreFile = tempSubDir.toPath().resolve("public.jks").toFile();
-    certUtils.createPublicCert(publicKeyStoreFile, "test", rootKeyStoreFile.getAbsolutePath(),
+    Path tempSubDir = tempDir.resolve(testName());
+    if (!Files.isDirectory(tempSubDir)) {
+      Files.createDirectories(tempSubDir);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier